extensions/mssql/src/objectExplorerNodeProvider/hdfsCommands.ts

```diff
@@ -322,6 +331,17 @@ export class PreviewFileCommand extends ProgressCommand {
 		}
 	}
 
+	private async showNotebookDocument(fileName: string, connectionProfile?: azdata.IConnectionProfile,
+		initialContent?: string): Promise<azdata.nb.NotebookEditor> {
```

Comment:
nitpick: does this need to be on a 2nd line? If so, we usually put the `): Promise<azdata.nb.NotebookEditor> {` part on its own 3rd line so that indentation looks "correct". Thanks!

Reply:
fixed.